integration_tests/tests/vault/set_fees.rs

```diff
@@ -0,0 +1,430 @@
+#[cfg(test)]
+mod tests {
+    use jito_vault_core::{config::Config, vault::Vault};
+    use jito_vault_sdk::{error::VaultError, instruction::VaultAdminRole};
+    use solana_sdk::signature::{Keypair, Signer};
+
+    use crate::fixtures::{
+        fixture::TestBuilder,
+        vault_client::{assert_vault_error, VaultRoot},
+    };
+
+    #[tokio::test]
+    async fn test_initialize_vault_with_bad_fees() {
+        let fixture = TestBuilder::new().await;
+
+        let mut vault_program_client = fixture.vault_program_client();
+
+        let _config_admin = vault_program_client.setup_config().await.unwrap();
+        let vault_base = Keypair::new();
+
+        let vault_pubkey =
+            Vault::find_program_address(&jito_vault_program::id(), &vault_base.pubkey()).0;
+
+        let lrt_mint = Keypair::new();
+        let vault_admin = Keypair::new();
+        let token_mint = Keypair::new();
+
+        vault_program_client
```

Comment:
you can use vault_program_client.setup_config_and_vault() instead of setting up a bunch of this context. but if not, should remove the _airdrop and _create_token_mint on these as they're public facing

Reply:
Done

Reply:
Done